source/concurrent/Promise.ooc

```diff
@@ -33,6 +33,13 @@ Promise: abstract class {
 	start: static func (action: Func) -> This {
 		_ThreadPromise new(action)
 	}
+	createEmpty: static func -> This { _EmptyPromise new() }
```

Comment:
From [Wiktionary](https://en.wiktionary.org/wiki/empty_promise):
```
empty promise ‎(plural empty promises)
(idiomatic) A promise that is either not going to be carried out, worthless or meaningless.
```
I guess it works. But it would be nicer with a static `empty` property which returns an `_EmptyPromise` instance, that would be more in line with the style in similar situations.